src/upx.mk

```diff
@@ -47,7 +47,9 @@ define $(PKG)_BUILD_$(BUILD)
         'CC=$(BUILD_CC)' \
         'PKG_CONFIG=$(PREFIX)/$(BUILD)/bin/pkgconf' \
         'LIBS=-L$(PREFIX)/$(BUILD)/lib -lucl -lz' \
-        $(shell [ `uname -s` == Darwin ] && echo "CXXFLAGS='-Wno-error=unused-local-typedef'") \
+        $(shell [ `uname -s` == Darwin ] && \
+            echo "CXXFLAGS=-Wno-error=unused-local-typedefs -Wno-error=misleading-indentation" || \
+            echo "CXXFLAGS=-Wno-error=misleading-indentation") \
```

Comment:
Hi Tony, This seems to have a side effect. Using gcc (Ubuntu 5.4.0-6ubuntu1~16.04.2) 5.4.0 20160609:

    g++ -DWITH_LZMA=0x920 -I/home/brand/projects/mxe/mxe/tmp-upx-x86_64-unknown-linux-gnu/upx-3.91-src/ucl-1.03/include -I/home/brand/projects/mxe/mxe/tmp-upx-x86_64-unknown-linux-gnu/upx-3.91-src/lzma -Wno-error=misleading-indentation -o c_file.o -c c_file.cpp
    cc1plus: error: -Werror=misleading-indentation: no option -Wmisleading-indentation